showcase/src/app/demo/logout/logout.component.html

```diff
@@ -0,0 +1,3 @@
+<example-viewer [extensions]="['HTML']" filesPath="logout/logout" exampleTitle="Logout button">
+	<stark-app-logout icon="power"></stark-app-logout>
+</example-viewer>
```

Comment:
The same remark as @SuperITMan, you can provide 2 examples, one with no `icon` provided and the other one with an `icon` provided